Trappist/src/Promact.Trappist.Web/Startup.cs

```diff
@@ -123,14 +107,6 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             }
             context.Database.Migrate();
             context.Seed();
-            #region Auto Mapper Configuration
-            Mapper.Initialize(cfg =>
```

Comment:
As per the current model schema I don't need automapper,as I am not mapping any property of any model class to any application class in the server side, therefore I removed automapper configurations.